internal/storage/sqlite/sqlite.go

```diff
@@ -0,0 +1,159 @@
+package sqlite
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/shmel1k/qumomf/internal/vshard/orchestrator"
+
+	"github.com/shmel1k/qumomf/internal/vshard"
+
+	"github.com/shmel1k/qumomf/internal/storage"
+)
+
+const (
+	querySaveSnapshot = `INSERT INTO snapshots(cluster_name, created_at, data) 
+							VALUES(?, ?, ?)`
+	querySaveRecoveries = `INSERT INTO recoveries(cluster_name, created_at, data) 
```

Comment:
I am afraid that it is not a good idea to save all cluster snapshots, especially without expiration. It is enough to store the last one, isn't it?

Reply:
changed the logic to update the current snapshot